code-challenges.js

```diff
@@ -58,12 +95,23 @@ const givenValue1 = 42
 const givenValue2 = 10
 // Expected output: 8
 
+        console.log(numberOfConnections.lastIndexOf())
+
 // --------------------5) Create the code that will sort all the numbers in an array from largest to smallest. Use the test variables provided below.
 
 // Pseudo code:
+//use .sort to modify the array.
+//reverse (a,b) to change it to largest to smallest
```

Comment:
There should definitely be more explanation here. This pseudo code doesn't show that you understand what's happening in the code, so make sure to have a full explanation in future pseudo code.